imperial_coldfront_plugin/gpfs_client.py

```diff
@@ -293,12 +293,36 @@ def retrieve_quota_usage(
         ]  # denotes the "Number of files in usage": Number of inodes.
 
         retrieved_data = {
-            "block_usage": block_usage,
+            "block_usage_gb": block_usage / 1024**2,
             "files_usage": files_usage,
         }
 
         return retrieved_data
 
+    @json
+    @get("filesystems/{filesystemName}/quotas")
+    def _retrieve_all_fileset_quotas(
+        self,
+        filesystemName: str,
+    ) -> requests.Response:
+        """Method (private) to retrieve the quotas of a filesystem."""
+
+    def retrieve_all_fileset_usages(self, filesystem_name: str):
+        """Get the quotas for all filesets.
+
+        Arguments:
+            filesystem_name: Name of the filesystem to retrieve the quota usage from.
+        """
+        data = self._retrieve_all_fileset_quotas(filesystem_name).json()
+        return {
+            quota["objectName"]: {
+                "files": quota["filesUsage"],
+                "block_gb": quota["blockUsage"] / 1024**2,
```

Comment:
I would suggest to be consistent with the naming of objects that represent the same things. Above you use `files_usage` and `block_usage_gb`, so here we should use the same names. Or the other way around and change the code above. Either way works fine as long as there is consistency.

Reply:
Good point. I've standardised.